### PR TITLE
LSR-1027 Liferay Screens Xamarin: ForgotPasswordScreenlet binding

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

File: xamarin/Framework/LiferayScreens.iOS/Themes/Default/ForgotPasswordView_default.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        void OnStartInteraction();
+
+        // -(void)onFinishInteraction:(id _Nullable)result error:(NSError * _Nullable)error;
+        [Export("onFinishInteraction:error:")]
+        void OnFinishInteraction([NullAllowed] NSObject result, [NullAllowed] NSError error);
+
+        // -(id<ProgressPresenter> _Nonnull)createProgressPresenter __attribute__((warn_unused_result));
+        //[Export("createProgressPresenter")]
+        //[Verify(MethodToProperty)]
+        //ProgressPresenter CreateProgressPresenter { get; }
```
Comment (victorg1991) [inline xamarin/Framework/LiferayScreens.iOS/Themes/Default/ForgotPasswordView_default.cs]: why is this commented?

Comment (dgarciasarai) [inline xamarin/Framework/LiferayScreens.iOS/Themes/Default/ForgotPasswordView_default.cs]: Because ProgressPresenter class is not created yet :)

Comment (victorg1991) [inline xamarin/Framework/LiferayScreens.iOS/Themes/Default/ForgotPasswordView_default.cs]: 👍

**Comment by dgarciasarai** [comment]
👍

**Pull request merged**